### ISSUE TITLE
Is there a way to transcribe multiple audio files asynchronously/parallel with whisperX?

### ISSUE THREAD

**Issue by imc-db** [m-bain/whisperX#861]
Is there a way to transcribe multiple audio files asynchronously/parallel with whisperX?

**Comment by 3manifold** [comment]
That's an interesting task; I can initiate that given that it is considered as a high priority feature by the repository authors. What are the chances that this is going to be merged if there is a pull request for this feature?

**Comment by jonwomack** [comment]
Temporary workaround I've been using:

`ls /path/to/your/folder/* | xargs -n 1 -P <num_parallel> bash -c 'whisperx "$0" --language en`

**Comment by Barabazs** [comment]
Yes it's possible, but not natively as this would considerably expand the scope of the project.
